test/Ocelot.Benchmarks/HeavyRoutesStartupBenchmark.cs

```diff
@@ -0,0 +1,162 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Ocelot.Configuration.File;
+using Ocelot.DependencyInjection;
+using Ocelot.Middleware;
+using System.Net.Sockets;
+using System.Net;
+using Iced.Intel;
+using Microsoft.AspNetCore.Routing;
+using System.Runtime.Intrinsics.X86;
+namespace Ocelot.Benchmarks
+{
+    [Config(typeof(HeavyRoutesStartupBenchmark))]
+    public class HeavyRoutesStartupBenchmark : ManualConfig
+    {
+        private IWebHost _ocelot;
+
+        public HeavyRoutesStartupBenchmark()
+        {
+            AddColumn(StatisticColumn.AllStatistics);
+            AddDiagnoser(MemoryDiagnoser.Default);
+            AddValidator(BaselineValidator.FailOnError);
+        }
+
+        [GlobalSetup]
+        public void SetUp()
+        {
+            //Setting up for more than 600 routes in different files
+            CreateOcelotConfigFile(0,100,"ocelot.json");
+            CreateOcelotConfigFile(101, 500, "ocelot.second.json");
```

Comment:
1st step: We need to have some routes in 7+ files
2nd step: We add new file with 80+ routes

We need to split into 2 scenarios
- new file routes are new ones
- new file routes override routes in old files